monitor/internal/docker/monitor.go

```diff
@@ -654,14 +657,27 @@ func (d *DockerMonitor) setupDockerDaemon() (err error) {
 func (d *DockerMonitor) waitForDockerDaemon(ctx context.Context) (err error) {
 
 	done := make(chan bool)
-	go func() {
-		for errg := d.setupDockerDaemon(); errg != nil; {
+	go func(gctx context.Context) {
+
+		for {
+			errg := d.setupDockerDaemon()
+			if errg == nil {
+				if err := d.initMonitor(gctx); err != nil {
+					zap.L().Debug("Unable to init monitor")
+				}
+				break
```

Comment:
I think this needs to be an error, so users can see that even though the docker daemon is there, we were never able to start it

Reply:
Done